documentation/modules/upgrade-notes-2.9.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="mtv-upgrade-2-9-0_{context}"]
+= Upgrade notes {project-short} 2.9.0
+
+To upgrade to {project-full} 2.9.0, you need to follow the manual upgrade process.
+
+The option to automatically upgrade has been disabled.
```

Comment:
`The option to automatically upgrade from previous versions Y version, such as 2.8, has been disabled.` wdyt?